_posts/2016-06-22-metronome-practice.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: post
+title: "Simple Ways to Rule Rhythm"
+author: "Hayley MacLeod"
+categories: practice
+header: /assets/img/2016-07-14/metronome.png
+---
+
+Rhythm is often under-rated both in practice and teaching. A strong sense of rhythm is one of the distinguishing features of a great musician. In a performance, inconsistent rhythms can lead to feelings of unease. What’s the fastest way to rule rhythm? __The metronome__.
+
+## 4 Ways to 'Level Up' Your Rhythm Game
+
+![](/assets/img/2016-07-14/tambourine.jpg){:width="75%"}
+
+#### Finding the Beat
+
+The beat of a song is similar to the tick-tock of a clock. Finding the beat in a song is like listening for the sound of a clock within a room. You may not recognize it until you focus on the sound. Listen to a [song](https://www.youtube.com/watch?v=DohRa9lsx0Q&list=PLK9Sc5q_4K6aNajVLKtkaAB1JGmKyccf2) and find the beat. Clap your hands, snap your fingers or tap your toes.
+
+Maintain following the beat for the entire song. Alternate between clapping, snapping, tapping or verbalizing the beat.
+
+*Exercise 1*
+
+- The next time you are practicing, set the [metronome](https://pitchplay.io) to 60.
+- Tap, clap, snap or verbalize the beat. Continue until you feel that the sound of the metronome fades and your accuracy has improved.
+- Try speeding up or slowing down the metronome.
+
+#### Internalizing the Beat
+
+When you have internalized a beat, you no longer have to rely on your toes or fingers to keep things at a steady pulse.  Internalizing the beat is important. It ensures that you will be able to maintain a consistent tempo.
```

Comment:
Beat internalization should be introduced as a concept before referencing it here. Assumption that the user knows that this is.